### PR TITLE
docs: Extending Cube.js

### PR DESCRIPTION
**Pull request by keydunov**

### CODE REVIEW

File: docs/content/Configuration/Extending-Cubejs.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+title: Extending Cube.js
+permalink: /configuration/extending-cubejs
+category: Configuration
+menuOrder: 5
+---
+
+You can install third-party Node.js packages to use inside `cube.js`
+configuration file. Commonly it's used for instrumentation or custom
+authentication logic.
```
Comment (hassankhan) [inline docs/content/Configuration/Extending-Cubejs.md:10]:
```suggestion
For some advanced use-cases like instrumentation, or custom
authentication, Cube.js can be extended by installing third-party
Node.js packages and using them in the [`cube.js` configuration file][ref-config].

[ref-config]: /config
```

File: docs/content/Configuration/Extending-Cubejs.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+## Example: sending Cube.js logs to Loggly
+
+The example below shows how to use Node.js Loggly client to collect and send
```
Comment (hassankhan) [inline docs/content/Configuration/Extending-Cubejs.md:14]:
```suggestion
The example below shows how to use the Node.js Loggly client to collect and send
```

File: docs/content/Configuration/Extending-Cubejs.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+logs from Cube.js.
+
+First, you'd need to install the third-party library with [NPM](https://www.npmjs.com/). In our example
+we're going to use
+[winston-loggly-bulk](https://github.com/loggly/winston-loggly-bulk) to collect
+and send logs to Loggly. You can install it with the following command.
```
Comment (hassankhan) [inline docs/content/Configuration/Extending-Cubejs.md:20]:
```suggestion
and send logs to Loggly. You can install it with the following command:
```

File: docs/content/Configuration/Extending-Cubejs.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+| When installing custom Node.js packages for Cube.js running in Docker container, make sure you correctly mount the folder with `node_modules` subfolder.
+| ```bash
+| $ docker run -d \
+|   -v ~/my-cubejs-project:/cube/conf
+|   cubejs/cube
+| ```
+| If you need to use third-party Node.js packages with native extensions, you'd need to build your own Docker image.
+
+
+Now we can require and use `winston-loggly-bulk` library inside the `cube.js` file.
```
Comment (hassankhan) [inline docs/content/Configuration/Extending-Cubejs.md:36]:
```suggestion
Now we can require and use `winston-loggly-bulk` library inside `cube.js`:
```

**Review by hassankhan — COMMENTED** [review]
Looks great 👍 Added some suggestions

Do let me know before merging, I'd like to run Prettier on the file first 😄

**Comment by keydunov** [comment]
Thanks, @hassankhan. I've committed your proposed changes. It is now ready for merge.

**Pull request merged**